app/main/routes.py

```diff
@@ -659,7 +660,11 @@ def download_record(record_id: uuid.UUID):
             "Content-Disposition": "attachment;filename=" + download_filename
         },
     )
-
+    keycloak_openid = get_keycloak_instance_from_flask_config()
+    decoded_access_token = keycloak_openid.introspect(session["access_token"])
+    user_id = decoded_access_token["sub"]
+    log_download_data = {"user_id": user_id, "file": key}
+    print(json.dumps(log_download_data))
```

Comment:
can we replace this with use of the configured logger;

with appropriate log level;

```
```python
current_app.logger.info('Some info message')
current_app.logger.debug('Some debug message')
current_app.logger.warning('Some warning message')
current_app.logger.error('Some error message')
```
```

Reply:
updated code to use current_app.logger.info as requested